src/it/resources/org/checkstyle/suppressionxpathfilter/illegaltoken/SuppressionXpathRegressionIllegalToken2.java

```diff
@@ -0,0 +1,5 @@
+package org.checkstyle.suppressionxpathfilter.illegaltoken;
+
+public class SuppressionXpathRegressionIllegalToken2 {
+  public native void myTest(); //warn
```

Comment:
Please read check documentation and update test module config.

Reply:
Done